h/static/scripts/directive/test/annotation-test.js

```diff
@@ -1502,27 +1472,36 @@ describe('annotation.js', function() {
 
       it('moves new annotations to the focused group', function() {
         var annotation = defaultAnnotation();
+        // id must be null so that AnnotationController considers this a new
+        // annotation.
         annotation.id = null;
```

Comment:
We're mixing up use of null and undefined in difference places, I need to do a little more background reading but I think it would make sense to recommend exclusively using one or the other in the style guide in future.